src/chrome/content/rules/Eloqua.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://images.marketing.eloqua.com/ => https://s33.t.eloqua.com/: Too many redirects while fetching 'https://s33.t.eloqua.com/'
+Fetch error: http://now.eloqua.com/ => https://secure.eloqua.com/: (51, "SSL: no alternative certificate subject name matches target host name 'now.eloqua.com'")
```

Comment:
This error is actually about fetching the insecure version, http://now.eloqua.com, which redirects to https://now.eloqua.com/FormsLogin.aspx, which has an invalid cert.

Do you know of any resources hosted on the insecure version of now.eloqua.com that successfully load and should be redirected to the secure version?